merfi/backends/gpg.py

```diff
@@ -58,3 +72,30 @@ def sign_release(self, path):
             logger.info('signing: %s' % path)
             util.run(detached)
             util.run(clearsign)
+
+    def sign_repomd(self, path):
+        """ Sign a Yum repomd.xml file. """
+        if merfi.config.get('check'):
+            new_md_path = path + '.asc'
+            logger.info('[CHECKMODE] signing: %s' % path)
+            logger.info('[CHECKMODE] signed: %s' % new_md_path)
+            return
+        os.chdir(os.path.dirname(path))
```

Comment:
this line and the `detached` command should be at the beginning so that when `CHECKMODE` is on, it is closer to the truth on what it would be doing (changing directories for one), and reporting the actual `path` that would be used

Reply:
good suggestion, I've fixed this and amended this PR.